src/components/input/input.js

```diff
@@ -138,6 +140,12 @@ function labelDirective() {
  *       <div ng-message="md-maxlength">That's too long!</div>
  *     </div>
  *   </md-input-container>
+ *   <md-input-container>
+ *     <input aria-label='title' ng-model='title'>
```

Comment:
I'd be surprised if anyone used `md-input-container` without a placeholder *or* label element. Maybe we should show it with placeholder and aria-label?